<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Стеганография</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="modeLayout">
      <item>
       <widget class="QRadioButton" name="radioEmbed">
        <property name="text">
         <string>Встраивание</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioExtract">
        <property name="text">
         <string>Извлечение</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="methodLayout">
      <item>
       <widget class="QRadioButton" name="radioMethodPVD">
        <property name="text">
         <string>PVD</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioLSB">
        <property name="text">
         <string>LSB-DCT</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="buttonLoadImage">
      <property name="text">
       <string>Загрузить изображение</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLabel" name="labelOriginal">
        <property name="text">
         <string>Исходное изображение</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelResult">
        <property name="text">
         <string>Результат</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="labelMessage">
      <property name="text">
       <string>Сообщение:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineMessage"/>
    </item>
    <item>
     <widget class="QLabel" name="labelBitCount">
      <property name="text">
       <string>Количество бит:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinNumBits">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1000000</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonProcess">
      <property name="text">
       <string>Встроить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textOutput">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
